src/main/java/com/couchbase/lite/Database.java

```diff
@@ -74,6 +74,10 @@
     // When this many changes pile up in _changesToNotify, start removing their bodies to save RAM
     private static final int MANY_CHANGES_TO_NOTIFY = 5000;
 
+    private static final String DEFAULT_PBKDF2_KEY_SALT = "Salty McNaCl";
```

Comment:
It's just a default salt used for deriving PBDKF2. The is also the same as the iOS implementation. I think NaCl is Sodium Chloride. I don't know what is Mc.